packages/home-spa/src/styles/_mixin.scss

```diff
@@ -0,0 +1,14 @@
+@mixin button {
+    display: inline-block;
+    background: none;
+    border: none;
+    margin: 0;
+    padding: 1rem 2rem;
+    font-weight: 600;
+    cursor: pointer;
+    border-radius: 3px;
+    text-decoration: none;
+    text-align: center;
+    white-space: nowrap;
+    vertical-align: middle;
+}
```

Comment:
Can things like these (colors, styles, mixins, components) be please moved to a new packages/common directory so that all apps and microservices can reuse these?

Reply:
The packages/common directory is a good option and the only flaw I see is that it would be too vague as to what all is common, is it just for SPAs or is it also for microservices? Hence, for now, it is in the same home-spa package.